pitmanagementsystem/workshop/src/renderer/src/App.tsx

```diff
@@ -1,18 +1,193 @@
-import { Box } from '@mui/material';
+import {
+  AppBar,
+  CSSObject,
+  Theme,
+  useTheme,
+  Box,
+  Button,
+  Drawer,
+  Icon,
+  IconButton,
+  Toolbar,
+  Typography,
+  styled,
+  AppBarProps,
+  Divider,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+} from '@mui/material';
 import React, { ReactElement } from 'react';
-import { Routes, Route } from 'react-router-dom';
+import { Routes, Route, Link } from 'react-router-dom';
 import { ExampleTitle } from './components/ExampleTitle';
+import SystemMeltDownGearLogo from './assets/SystemMeltdownGearLogo.png';
+import InboxIcon from '@mui/icons-material/MoveToInbox';
+import MailIcon from '@mui/icons-material/Mail';
+import { Star } from '@mui/icons-material';
+import { Dashboard } from './components/Dashboard';
+import { Inventory } from './components/Inventory';
+
+const drawerWidth = 240;
+
+const openedMixin = (theme: Theme): CSSObject => ({
+  width: drawerWidth,
+  transition: theme.transitions.create('width', {
+    easing: theme.transitions.easing.sharp,
+    duration: theme.transitions.duration.enteringScreen,
+  }),
+  overflowX: 'hidden',
+});
+
+const closedMixin = (theme: Theme): CSSObject => ({
+  transition: theme.transitions.create('width', {
+    easing: theme.transitions.easing.sharp,
+    duration: theme.transitions.duration.leavingScreen,
+  }),
+  overflowX: 'hidden',
+  width: `calc(${theme.spacing(7)} + 1px)`,
+  [theme.breakpoints.up('sm')]: {
+    width: `calc(${theme.spacing(8)} + 1px)`,
+  },
+});
+
+const DrawerHeader = styled('div')(({ theme }) => ({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'flex-end',
+  padding: theme.spacing(0, 1),
+  // necessary for content to be below app bar
+  ...theme.mixins.toolbar,
+}));
+
+interface Custom_AppBarProps extends AppBarProps {
```

Comment:
Use `type` over `interface`. I am pretty sure it is stricter. I think that's what my work guy said.